crates/forge/tests/it/fuzz.rs

```diff
@@ -160,7 +160,7 @@ async fn test_scrape_bytecode() {
     let filter = Filter::new(".*", ".*", ".*fuzz/FuzzScrapeBytecode.t.sol");
     let mut runner = TEST_DATA_DEFAULT.runner();
     runner.test_options.fuzz.runs = 2000;
-    runner.test_options.fuzz.seed = Some(U256::from(100u32));
+    runner.test_options.fuzz.seed = Some(U256::from(120u32));
```

Comment:
Why is this needed?

Reply:
Re: https://github.com/foundry-rs/foundry/pull/8938#issuecomment-2368871161

cc @grandizzy

Reply:
when we fuzz we use test contract bytecode which changed with adding the new cheatcode so there are now different fuzzed inputs, had similar here: https://github.com/foundry-rs/foundry/pull/8882/files#diff-d6c9e2f7cec6e686d5693fd6c6bd6400e4f617e1dea2695df29cab51549ba797R163
ref https://github.com/foundry-rs/foundry/issues/8115

Reply:
I don't see how changing an interface (Vm) change bytecode of an unrelated contract

Reply:
the issue can be reproduced in master by appending a line like `function writeToml() external;` in `testdata/cheats/Vm.sol` and `cargo test test_scrape_bytecode` will start failing. It can make it pass if bumping fuzz runs from 2000 to 5000. let me create a simpler repro to expose such

Reply:
I made some more tests with a simpler repro and figured out that happens because test has target contract in test contract, so they get inline, if they're split then issue is not reproducible and test pass without changing the seed, here's a draft PR https://github.com/foundry-rs/foundry/pull/8953
@DaniPopes

Reply:
this change is no longer needed, the flaky test was fixed